### PR TITLE
[fix bug 1431795] Add wait face landing page variation with video

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new variation of the Wait Face campaign landing page with an embedded video at `/firefox/new/?xv=2xfaster` (en-US only).

~**Do not merge** for now until I get approval from Fabio.~

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1431795

## Testing
Demo: https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=2xfaster

- Ensure video embed functions correctly and interaction GA events fire as expected.
- Test responsiveness.

### CODE REVIEW

**Comment by alexgibson** [comment]
Still waiting on some analytics confirmation, but this should be good for code review.

**Comment by alexgibson** [comment]
@jpetto r? I believe this is supposed to be one of those quick turnaround page variation things, (although no one has really stated that in the bug).

File: media/js/firefox/new/wait-face-scene1-video.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    function trackVideoInteraction(title, state) {
+        window.dataLayer.push({
+            'event': 'video-interaction',
+            'videoTitle': title,
+            'interaction': state
+        });
+    }
+
+    function initVideoInteractionTracking() {
+        var videos = document.querySelectorAll('video');
```
Comment (jpetto) [inline media/js/firefox/new/wait-face-scene1-video.js]: Any reason not to give the `<video>` an `id` and skip the `for` loop below?

Comment (alexgibson) [reply media/js/firefox/new/wait-face-scene1-video.js]: Bah, yes - this was hastened copy/pasta from another page 👍

**Comment by alexgibson** [comment]
@jpetto thanks for the review, updated

**Comment by jpetto** [comment]
Squasherino?

**Comment by alexgibson** [comment]
Squashed, but let's wait for Peter to give GA the ok before we merge. Thanks @jpetto!

**Comment by alexgibson** [comment]
@jpetto I added a new commit that adds some UTM params to the blog links. Other than this, I think we're now good to merge after another quick r?

**Comment by jpetto** [comment]
r+!

**Comment by alexgibson** [comment]
@jpetto squashed, merge at will! :shipit:

**Pull request merged**